src/stories/NumberInput.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import { useArgs } from '@storybook/client-api';
+import { TiSortNumerically } from 'react-icons/ti';
+
+import NumberInput, { NumberInputProps } from '#components/NumberInput';
+
+export default {
+    title: 'Input/NumberInput',
+    component: NumberInput,
+    argTypes: {},
+};
+
+const Template = (args: NumberInputProps) => {
+    const [{ value }, updateArgs] = useArgs();
+
+    const handleChange = (e) => {
+        updateArgs({ value: e });
+    };
+
+    console.log('outside', value);
```

Comment:
We should remove this may be?

Reply:
https://www.youtube.com/watch?v=S9RVS8cjNN0